test/FunctionalTests/Microsoft.OData.Core.Tests/Json/MockODataUtf8JsonWriterFactory.cs

```diff
@@ -11,11 +11,11 @@
 
 namespace Microsoft.OData.Tests.Json
 {
-    internal sealed class MockStreamBasedJsonWriterFactory : IStreamBasedJsonWriterFactory
+    internal sealed class MockODataUtf8JsonWriterFactory : IJsonWriterFactory

```

Comment:
```suggestion
    internal sealed class MockJsonWriterFactory : IJsonWriterFactory

```

Since this class takes any `IJsonWriter` implementation, I don't think it makes sense to couple the name to `ODataUtf8JsonWriter`.